File: XBRL/xbrl_parse.rb
require "rexml/document"

doc = REXML::Document.new File.new(ARGV[0])

values = doc.get_elements("//jpfr-t-cte:OperatingIncome[@contextRef='CurrentYearConsolidatedDuration']")

values.each do |n|
	puts n.text
end

File: XBRL/xbrl_parse.js

var doc = WScript.CreateObject("MSXML2.DOMDocument");

doc.async = false;
doc.setProperty("SelectionLanguage", "XPath");
//XMLñºëOãÛä‘ÇÃê›íË
doc.setProperty("SelectionNamespaces", "xmlns:jpfr-t-cte='http://info.edinet-fsa.go.jp/jp/fr/gaap/t/cte/2010-03-11'");

doc.load(WScript.Arguments.Item(0));

var values = doc.selectNodes("//jpfr-t-cte:OperatingIncome[@contextRef='CurrentYearConsolidatedDuration']");

for (var i = 0; i < values.length; i++) {
	WScript.Echo(values.item(i).text);
}
